velociraptor/autoplotter/objects.py

```diff
@@ -1217,6 +1208,17 @@ def create_plots(
                 if debug:
                     import sys, traceback
 
+                    _, _, exc_traceback = sys.exc_info()
+                    print("Traceback:")
```

Comment:
Sure, but this is just copying the bit of code just below, so this is not really part of this PR.

Reply:
Also: the traceback functions that are used in this block and the other blocks explicitly print to `stdout`. So that does not seem very consistent with your comment.

Reply:
Yeah they do but I think that may have been a mistake. I think we want the errors to go to stderr, and then the users can redirect as they wish.